index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap' rel='stylesheet' />
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Portfolio</title>
+
+</head>
+
+<body>
```

Comment:
- [ ] You really have a nice design 👏 . However, there's a slight adjustment you need t make on the right margin adjacent to the screen. there seems to be a white space left which ideally should fit into the screen. 
- I've attached a comparison picture for your review. Making this change gives a better user experience 👍 

## Your view 👍 
![image](https://user-images.githubusercontent.com/50990925/219283695-2e266462-2db9-41b1-804f-c677f48a465b.png)

## Figma Template:
![image](https://user-images.githubusercontent.com/50990925/219283770-d6981367-4e82-4d5d-8f02-d536debb4dfa.png)